lcdb_test_data/Snakefile

```diff
@@ -57,32 +72,102 @@ rule flux:
                 READ_NUMBER	5000
                 ERR_FILE	76
                 FASTA	YES
+                PAIRED_END	true
+                UNIQUE_IDS	true
                 SEED	{sample_int}
                 """.format(**locals())))
         shell(
             'flux-simulator --force -t simulator -x -l -s -p {fout.name} &> {log}')
 
+        # flux-simulator creates an interleaved fastq. So we deinterleave, idea
+        # from https://gist.github.com/nathanhaigh/3521724
+        #
+        # Note that if UNIQUE_IDS is false (by default), read names end with
+        # "S/2" or "A/1" and "S" or "A" indicate "sense" or "antisense". When
+        # UNIQUE_IDS is true (as we use here), they only have /1 or /2, and
+        # read 1 is always sense while read 2 is antisense.
+        #
+        # Read more by checking `flux-simulator --printParameters`.
+        #
+        fastq = 'samples/{sample}/{sample}.fastq'.format(sample=wildcards.sample)
+        shell(
+            'cat {fastq} '
+            '| paste - - - - - - - - '
+            '| tee '
+            '>(cut -f 1-4 | tr "\t" "\n" | gzip -c > {output.fastq_R1}) '
+            '| cut -f 5-8 | tr "\t" "\n" | gzip -c > {output.fastq_R2}'
+        )
+
+
+# ----------------------------------------------------------------------------
+# HISAT2 index
 rule hisat_index:
     input: rules.prep_fasta.output
     output: expand('seq/2L.{n}.ht2', n=range(1,8))
     params: index='seq/2L'
+    log: 'seq/2L.ht2.log'
     shell:
-        'hisat2-build {input} {params.index}'
+        'hisat2-build {input} {params.index} &> {log}'
 
```

Comment:
My intention was to use these data for testing wrappers, so I wanted to keep it as simple as possible rather than introduce possibly circular dependencies.